fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1850,39 +1909,42 @@ public static OnlineIndexer forRecordStoreAndIndex(@Nonnull FDBRecordStore recor
         /**
          * Only build if the index is disabled.
          */
-        BUILD_IF_DISABLED(false),
+        BUILD_IF_DISABLED,
         /**
-         * Build if the index is disabled; Continue build if the index is write-only.
+         * Build if the index is disabled; Continue build if the index is write-only - using the previous
+         * method that was used.
          * <p>
          * Recommended. This should be sufficient if current index data is not corrupted.
          * </p>
          */
-        BUILD_IF_DISABLED_CONTINUE_BUILD_IF_WRITE_ONLY(true),
+        BUILD_IF_DISABLED_CONTINUE_BUILD_IF_WRITE_ONLY,
```

Comment:
How is this related to `BUILD_IF_DISABLED_CONTINUE_BUILD_IF_WRITE_ONLY_ERROR_IF_POLICY_CHANGED` and `BUILD_IF_DISABLED_CONTINUE_BUILD_IF_WRITE_ONLY_REBUILD_IF_POLICY_CHANGED`? The comments should be updated in include this information.

Reply:
Updated the comment.